pkgs/development/python-modules/pywizlight/default.nix

```diff
@@ -7,27 +7,28 @@
 
 buildPythonPackage rec {
   pname = "pywizlight";
-  version = "0.4.1";
+  version = "0.4.4";
 
   src = fetchFromGitHub {
     owner = "sbidy";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0kyhyda28zbni9sjv6kvky6wlhqldl47niddgpbjsv5dlb9xvxns";
+    sha256 = "sha256-GweGgBnfHuGbsKxiHi976fgq/XRu4W47wAw15321Mo0=";
   };
 
   propagatedBuildInputs = [
     asyncio-dgram
     click
   ];
 
-  # no tests are present
+  # Tests requires network features (socketserver on localhost)
```

Comment:
And that fails? Local networking should not be an issue afaik.

Reply:
Some are failing...the test of the discovery feature and two tests for the timeout are not passing.